metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -163,15 +165,23 @@ private static CreatePublicationRequest createRequestWithDate(PublicationDate da
 
     private static Stream<Arguments> provideMetadataForTitle() {
         String title = "Title";
+        String longerTitle = "This is a longer title";
```

Comment:
Not a fan of ```var```?

Reply:
Never used it, currently following the precedent set in the class.